### ISSUE TITLE
Typo in DFO example. See the 'non-noisy DFO' results

### ISSUE THREAD

**Issue by caleb-hamilton** [numericalalgorithmsgroup/NAGPythonExamples#10]
The label currently says 'No noise' when it should say 'Noise level x, Derivative Free'

**Comment by caleb-hamilton** [comment]
With new pull request, I think I fixed the typos (and some bonus format issues). Once the pull request is merged, this issue can be closed.
